src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx

```diff
@@ -27,44 +25,39 @@ import { transformServiceHistoryEntryIntoTableRow } from '../../helpers';
 // Alert to show when a user does not appear to be a Veteran
 const NotAVeteranAlert = () => {
   return (
-    <AlertBox
-      isVisible
-      status="warning"
-      headline="We don’t seem to have your military records"
-      content={
-        <>
-          <p>
-            We’re sorry. We can’t match your information to our records. If you
-            think this is an error, please call the VA.gov help desk at{' '}
-            <Telephone contact={CONTACTS.HELP_DESK} /> (TTY:{' '}
-            <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
-            ). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. ET.
-          </p>
-          <p>
-            Or you can learn how to{' '}
-            <a
-              href="https://www.archives.gov/veterans/military-service-records/correct-service-records.html"
-              target="blank"
-              rel="noopener noreferrer"
-            >
-              update or correct your military service history
-            </a>
-            .
-          </p>
-        </>
-      }
-    />
+    <>
+      <va-alert status="warning">
+        <h2 slot="headline">We don’t seem to have your military records</h2>
+
+        <p>
+          We’re sorry. We can’t match your information to our records. If you
+          think this is an error, please call the VA.gov help desk at{' '}
+          <va-telephone contact={CONTACTS.HELP_DESK} /> (TTY:{' '}
+          <va-telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
```

Comment:
Probably not important for this PR, but I don't think the `pattern` prop is supported in the web component

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-telephone/va-telephone.tsx

Reply:
Adam is right - you can see the props the component supports [on its documentation page](https://design.va.gov/components/telephone#code-usage) as well as on the [storybook page](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default).

Reply:
That is amazing @bkjohnson ! Thanks for the tip.